### ISSUE TITLE
[Entry] Java Hashmap: Create the term entry `map.replace()`

### ISSUE THREAD

**Issue by dakshdeepHERE** [Codecademy/docs#4182]
### Reason/inspiration (optional)

We want to add the `.replace()` method as a new term entry for [HashMap](https://www.codecademy.com/resources/docs/java/hashmap). Provide the entry in a markdown file and place it under the file path `content/java/concepts/array-list/terms/subList/replace.md`

The entry should have the following:
– An introductory paragraph about what the method does.
– A syntax section with a pseudo code block explaining it.
– An example showing the method in use.

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by Sahillather002** [comment]
pls assign

**Comment by dakshdeepHERE** [comment]
@Sahillather002 You're assigned. Be sure to look at the documentation linked in the description and the [contribution guide](https://www.codecademy.com/resources/docs/contribution-guide) if you haven't already. 

Would this be your first official assignment? If so, how is it you find out about contributing to Codecademy Docs?

**Comment by SSwiniarski** [comment]
@Sahillather002 It's been two weeks. Are you still working on this issue? Note that in another week it will be unassigned due to inactivity.

**Comment by SSwiniarski** [comment]
@Sahillather002 this is being unassigned due to inactivity.

**Comment by jschikora** [comment]
@dakshdeepHERE Hi! I would do this because I'm currently doing Codecademy's Git course and a project of that course requires contributing to the docs.

**Comment by Aquacoder345** [comment]
Hello,
I would like to work on this entry.

**Comment by Great-Visions-Code** [comment]
@dakshdeepHERE @SSwiniarski Hi all! I am interested in taking this issue on.  I'm currently doing Codecademy's Apple Dev Course. This would be my first official assignment.

**Comment by Aquacoder345** [comment]
@dakshdeepHERE yes, i am still interested in taking this issue. It is for my Git/Github course i am taking.

**Comment by mshai1** [comment]
Hi @dakshdeepHERE can u please assign it to me?

**Comment by lynch2812** [comment]
Hi @dakshdeepHERE can i please be assigned to this issue? i would like to work on it.

**Comment by Aquacoder345** [comment]
@dakshdeepHERE has this issue been resolved or assigned to someone else?

**Comment by Arya-Winters** [comment]
Hello, 

I was wondering if this issue has been assigned or not? 

If not, may I be assigned to it? I am currently completing the Git&Github course on CodeAcademy and need to contribute to docs for part of it.

Thanks!

**Comment by Kmerica** [comment]
@dakshdeepHERE I am interested in taking on this issue.

**Comment by adrianna-yixin** [comment]
Hello @dakshdeepHERE, I am interested in taking this issue on to gain more experience contributing to the GitHub project 😸

**Comment by willphan1712000** [comment]
Hello, I am from Learn Git and Github Course on Codecademy. I am interested in this issue and I would like to take part in it.

**Comment by VinceIP** [comment]
I would like to be assigned to this issue as my first open source contribution.

**Comment by NoahHorners** [comment]
Hi @dakshdeepHERE, I am here to be tested and complete my Git Project on the Learn Git & GitHub course on this amazing website called Codecademy, I don't know if you have heard of it, but its some cool stuff. Anyways, I would be happy to be assigned to this issue.

**Comment by letsbuildanything** [comment]
@NoahHorners, Are you still interested in working on this issue?

**Comment by daniellelamb** [comment]
I'd like to be assigned this issue if it is still available. Thank you!

**Comment by letsbuildanything** [comment]
@daniellelamb You’re assigned :tada:  In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide).  After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions prior to approval.
Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by daniellelamb** [comment]
Hi! I am trying to finish my Git lesson on Codecademy. I requested this because I'm learning Javascript, and most of the other issues were with Python.

I'm now realizing that Java and Javascript are radically different. Is it possible to finish the "Git and Github, Part II" lesson if you haven't learned programming languages such as Python?

**Comment by daniellelamb** [comment]
Unless you are able to answer my previous question, please unassign me. Thanks!

**Comment by ishg-153** [comment]
Hey @daniellelamb! You are correct Java and JavaScript are both quite different. I suggest you look through the [available issues](https://github.com/Codecademy/docs/issues?q=is%3Aopen+is%3Aissue+-label%3A%22claimed%22), ask to be assigned to one that interests you and once assigned you can create a PR for that issue. 

I will go ahead and unassign this Issue from you.

**Comment by mamtawardhani** [comment]
@NoahHorners @VinceIP @willphan1712000 @Kmerica are any of is interested in taking up this issue?

**Comment by safi-mehrabi** [comment]
@dakshdeepHERE Hi, I'm interested in taking this issue on, please assign it to me as part of my Git course.

**Comment by mamtawardhani** [comment]
Hey @safi-mehrabi! you're assigned!

You can look through the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) to know more about contributing to Codecademy Docs. Check out the [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) to get help with creating your entry.

**Comment by ishg-153** [comment]
@safi-mehrabi It's been a week. How is the work on this entry coming along?

**Comment by ishg-153** [comment]
@safi-mehrabi you are being unassigned due to lack of activity.

**Comment by cigar-galaxy82** [comment]
I would love to work on this issue

**Comment by ishg-153** [comment]
@cigar-galaxy82 You're assigned!
